en/docs/learn/enabling-geolocation-based-statistics.md

```diff
@@ -3,11 +3,9 @@
 Follow the steps below to configure WSO2 IS Analytics Server to display
 the regions of the users authenticated though WSO2 Identity Server.
 
-1.  Download the geolocation data from
-    [here](https://svn.wso2.org/repos/wso2/people/tharindua/geodata/Geolocation%20Data.tar.gz)
+1.  Use the Geo Location dataset that you created [here](../../learn/creating-geo-location-data-set/) when configuring geo location based statistics.
```

Comment:
Better to say "Create a Geo Location dataset by following the guide given [here]"

@gomathyk Please verify

Reply:
Thanks. Addressed with https://github.com/wso2/docs-is/pull/2716/commits/ad42ac1b9834f3a3ac2e418711faf1986dae00db

Reply:
Updated the order with https://github.com/wso2/docs-is/pull/2716/commits/e984f4891204e792e64f06f501641fb304d8a9fe